tests/wicked/ipv6/sut/t09_managed_on_pl_ne64.pm

```diff
@@ -0,0 +1,58 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Managed on, prefix length != 64
+#   When radvd is set to managed on, prefix length != 64
+#   And I set up dynamic IP addresses for eth0 from legacy files
+#   And I let eth0 wait for a router announcement
+#   And I retrigger DHCP requests
+#   Then I should obtain an address by DHCPv4
+#   But I should not obtain an autonomous IPv6 address
+#   But I should obtain an address by DHCPv6
+#   And no address should be tentative
+#   And I should obtain a default IPv6 route
+#   And I should obtain a DNS server by DHCPv4
+#   But I should obtain a DNS server by DHCPv6
+#
+# https://github.com/openSUSE/wicked-testsuite/blob/master/features/wicked_6_ipv6.feature#L130
+#
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
```

Comment:
maybe we need to replace this with QAC ML or just add @mloviska  as forth person ? but of course not directly related to this PR

Reply:
makes sense, will do.